curriculum/templates/curriculum/lessonplan_find_share.html

```diff
@@ -0,0 +1,147 @@
+                        <div class="input-group-append">
+                            <button type="submit" class="btn btn-primary"><i class="fas fa-search"></i> Search</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="collapse {% if show_advanced_search_options %}show{% endif %} row" id="advancedSearchControls">
+                <div class="col mb-3">
+                    <div class="btn-group btn-group-toggle" data-toggle="buttons">
+                        <label class="btn btn-outline-primary {% if 'ES' in form.grade_level.value %}active{% endif %}">
+                            <input type="checkbox" name="grade_level" value="ES" autocomplete="off" {% if 'ES' in form.grade_level.value %}checked{% endif %} /> Elementary School
+                        </label>
+                        <label class="btn btn-outline-primary {% if 'MS' in form.grade_level.value %}active{% endif %}">
+                            <input type="checkbox" name="grade_level" value="MS" autocomplete="off" {% if 'MS' in form.grade_level.value %}checked{% endif %} /> Middle School
+                        </label>
+                        <label class="btn btn-outline-primary {% if 'HS' in form.grade_level.value %}active{% endif %}">
+                            <input type="checkbox" name="grade_level" value="HS" autocomplete="off" {% if 'HS' in form.grade_level.value %}checked{% endif %} /> High School
+                        </label>
+                        <label class="btn btn-outline-primary {% if 'U' in form.grade_level.value %}active{% endif %}">
+                            <input type="checkbox" name="grade_level" value="U" autocomplete="off" {% if 'U' in form.grade_level.value %}checked{% endif %} /> Post-secondary
+                        </label>
+                    </div>
+                </div>
+                <div class="col mb-3">
+                    <div class="btn-group-toggle" data-toggle="buttons">
+                        <label class="btn btn-outline-primary {% if form.web_only.value %}active{% endif %}" for="{{ form.web_only.id_for_label }}">
+                            <input type="checkbox" autocomplete="off" name="{{ form.web_only.name }}" {% if form.web_only.value %}checked{% endif %} /><i class="fab fa-chrome"></i> {{ form.web_only.label }}
+                        </label>
+                    </div>
+                </div>
+                <input type="hidden" name="user_id" value="{% if form.user_id.value %}{{ form.user_id.value }}{% endif %}" />
+            </div>
+        </form>
+        <div class="row my-5">
+            {% for lessonplan in object_list %}
+                <div class="col-4">
+                    <div class="curriculum mb-3">
+                        <img class = "img-responsive mb-3" 
+                            src="{% static 'img/computer_heart.jpg'%}" 
```

Comment:
What happens when there is no image associated with the curriculum? What happens if the images are different dimensions?

Reply:
**No Images Associated**
![Screen Shot 2019-06-26 at 8 46 47 PM](https://user-images.githubusercontent.com/23197955/60225159-8d30bd80-9853-11e9-8c9e-f5cf581082b9.png)
![Screen Shot 2019-06-26 at 8 52 12 PM](https://user-images.githubusercontent.com/23197955/60225318-48f1ed00-9854-11e9-9cad-24fca3cc3bcb.png)
**Different Dimensions**
![Screen Shot 2019-06-26 at 9 08 20 PM](https://user-images.githubusercontent.com/23197955/60225858-8f484b80-9856-11e9-9311-0bc4fa5838e3.png)

Examples for the questions, I agree it does look a bit weird though so we might want to either require photos and make them have to be a certain aspect ratio, or don't require but have a default photo to use

Reply:
I think there is a way in bootstrap have an image fill a certain space (sides may be cut off, but oh well). 

My opinion is that we don't do images for the MVP, since we still need to add the functionality in the upload workflow. We could make upload UI, database stuff, and curriculum page UI be its own issue.

Reply:
We can still have images for our own FirstByte curriculum on the home/about page, like the wireframes.

Reply:
I'm fine with doing no pictures, but up to everyone else

Reply:
I'm with jaedyn here, let's say no pictures for now. At least until we have a way for people to upload them along with their curriculum.